patches/applyPatches.sh

```diff
@@ -55,7 +56,7 @@ function applyPatch {
         showError "Faulty patch: $patch"
     fi
 
-    if [ "${clean_patch_dirs:-}" ]; then
+    if [ "$clean_patch_dirs" = 1 ]; then
```

Comment:
Yes and even better IMO as it avoids the quotes and is (more) clearly a (numeric) comparison:

```suggestion
    if ((clean_patch_dirs == 1)); then
```

See e.g. https://stackoverflow.com/a/18668580/1930508